docs/Users_Guide/overview.rst

```diff
@@ -349,7 +349,7 @@ METplus Components Python Requirements
       (metplotpy)
       <../generated/model_applications/s2s/TCGen_fcstGFSO_obsBDECKS_GDF_TDF.html>`_
   * - metpy
-    - 
+    - >=1.3.1
```

Comment:
This is listed as 1.3.1 in calcpy and plotpy nco_requirements.txt, but as 1.1.0 in calcpy requirements.txt.  Should we update the calcpy requirements.txt file?

Reply:
https://github.com/dtcenter/METcalcpy/issues/276